### PR TITLE
Add bottom inset to offset calculation

### PR DESCRIPTION
**Pull request by elfredpagan**
## Related Issue
Adds the adjusted bottom inset to the new offset calculation
## Motivation and Context

This resolves an issue where things are not correctly pinned during bounds changes.

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Review by brynbodayle — DISMISSED** [review]

File: MagazineLayout/Public/MagazineLayout.swift [hunk 1]
```diff
@@ -622,9 +622,15 @@ public final class MagazineLayout: UICollectionViewLayout {
     // If our layout direction is bottom to top we want to adjust scroll position relative to the
     // bottom
     if case .bottomToTop = verticalLayoutDirection {
+      let additionalOffsetAdjustment: CGFloat
+      if #available(iOS 11.0, tvOS 11.0, *) {
+        additionalOffsetAdjustment = currentCollectionView.adjustedContentInset.bottom
+      } else {
+        additionalOffsetAdjustment = 0.0
+      }
```
Comment (bryankeller) [inline MagazineLayout/Public/MagazineLayout.swift:630]:
```suggestion
```

File: MagazineLayout/Public/MagazineLayout.swift [hunk 2]
```diff
@@ -622,9 +622,15 @@ public final class MagazineLayout: UICollectionViewLayout {
...
       invalidationContext.contentOffsetAdjustment = CGPoint(
         x: 0.0,
-        y: currentCollectionView.bounds.height - newBounds.height)
+        y: currentCollectionView.bounds.height - newBounds.height + additionalOffsetAdjustment)
```
Comment (bryankeller) [inline MagazineLayout/Public/MagazineLayout.swift:633]:
```suggestion
        y: currentCollectionView.bounds.height - newBounds.height + contentInset.bottom)
```

**Review by bryankeller — COMMENTED** [review]
One suggestion, thanks!

**Comment by elfredpagan** [comment]
Accepted changes

**Review by bryankeller — APPROVED** [review]

**Pull request merged**